app/pages/order-task-list/template.test.js

```diff
@@ -9,109 +11,156 @@ const setup = {
+    };
+
+    harness.request(context, ($) => {
       const neworderPageDescription = $('[data-test-id="neworder-page-description"]');
       expect(neworderPageDescription.length).toEqual(1);
-      expect(neworderPageDescription.text().trim()).toEqual(neworderPageContext.description);
+      expect(neworderPageDescription.text().trim()).toEqual(neworderManifest.description);
     });
   }));
 
   it('should not render the order details', componentTester(setup, (harness) => {
-    harness.request(neworderPageContext, ($) => {
+    const context = {
+      orderId: 'neworder',
+    };
+
+    harness.request(context, ($) => {
       const neworderDescriptionTitle = $('[data-test-id="neworder-order-description-title"]');
       const neworderDescription = $('[data-test-id="neworder-order-description"]');
       expect(neworderDescriptionTitle.length).toEqual(0);
       expect(neworderDescription.length).toEqual(0);
     });
   }));
 
-  it('should render the neworder task-list component', componentTester(setup, (harness) => {
-    harness.request(neworderPageContext, ($) => {
+  it('should render the task-list component', componentTester(setup, (harness) => {
+    const context = {
+      orderId: 'neworder',
+      taskList: taskListManifest.taskList,
+    };
+
+    harness.request(context, ($) => {
       const taskListComponent = $('[data-test-id="task-list"]');
       expect(taskListComponent.length).toEqual(1);
     });
   }));
 
   it('should render the "Delete order" as a secondary and disabled button', componentTester(setup, (harness) => {
-    harness.request(neworderPageContext, ($) => {
+    const context = {
+      orderId: 'neworder',
+      deleteOrderButton: {
+        text: commonManifest.deleteOrderButton.text,
+        altText: commonManifest.deleteOrderButton.disabledAltText,
+        href: '#',
+        disabled: true,
+      },
+    };
+
+    harness.request(context, ($) => {
       const deleteOrderButton = $('[data-test-id="delete-order-button"]');
       expect(deleteOrderButton.length).toEqual(1);
-      expect(deleteOrderButton.text().trim()).toEqual(neworderPageContext.deleteOrderButton.text);
-      expect(deleteOrderButton.attr('aria-label')).toEqual(neworderPageContext.deleteOrderButton.altText);
+      expect(deleteOrderButton.text().trim()).toEqual(commonManifest.deleteOrderButton.text);
+      expect(deleteOrderButton.attr('aria-label')).toEqual(commonManifest.deleteOrderButton.disabledAltText);
       expect(deleteOrderButton.find('a').hasClass('nhsuk-button--secondary')).toEqual(true);
       expect(deleteOrderButton.find('a').hasClass('nhsuk-button--disabled')).toEqual(true);
     });
   }));
 
   it('should render the "Preview order summary" as a secondary and disabled button', componentTester(setup, (harness) => {
-    harness.request(neworderPageContext, ($) => {
+    const context = {
+      orderId: 'neworder',
+      previewOrderButton: {
+        text: commonManifest.previewOrderButton.text,
```

Comment:
same here with `...commonManifest.previewOrderButton`

Reply:
I had this originally but it also copies over the disabledAltText. I rather the context was concise with just what we need